src/common/transformers/GET.spec.js

```diff
@@ -0,0 +1,9 @@
+import expect from 'expect';
+
+import { get } from './GET';
+
+describe('GET', () => {
+    it("should return key's value from object", () => {
+        expect(get({ a: { aa: 1 }, b: 2 }, 'a.aa')).toEqual(1);
+    });
```

Comment:
test with no value, incorrect path, and no path too.

Reply:
I do not think it is useful to thoroughly test the lodash function, the value is elsewhere